### DISCUSSION TITLE
Sorting Environments by status

### DISCUSSION THREAD

**Discussion by austenstone** [community/community#12069] [Actions]
Should the Environments be sorted by status(Active, Pending, Inactive)?
I think it's alphabetical?
Got confused and realized my active env was hidden under "+3 environments"
![image](https://user-images.githubusercontent.com/22425467/155609532-07ad69aa-de21-46aa-bc0a-26bfb2aad35b.png)

**Comment by ethomson** [comment]
Thanks @austenstone - I'll pass this feedback along to the team so that as they continue to think about and iterate on our presentation that they have your feedback in mind.
